var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -32,7 +32,7 @@ class Neuron(CMakePackage):
     patch("fix_brew_py_18e97a2d.patch", when="@7.8.0c")
 
     version("develop", branch="master")
-    version("7.9.0a",  commit="acdd898", preferred=True)
+    version("7.9.0a",  commit="fc74b85", preferred=True)
```

Comment:
Did you delete the PR directory?

Reply:
you see - I am forgetting things :expressionless:! thanks!